app/controllers/events_controller.rb

```diff
@@ -56,9 +56,17 @@ def not_available_path
 
 private
 
+  def viewable_event?(event)
+    pending = event.status_id == pending_event_status_id
+    in_past = event.start_at.to_date < DateTime.now.utc.to_date
+    online_q_a = event.type_id == GetIntoTeachingApiClient::Constants::EVENT_TYPES["Online event"]
+
+    !pending && (!in_past || online_q_a)
```

Comment:
I think the `!`s make the final line difficult to parse, maybe it could be flipped a bit `!pending && (future || online_qa)`. I'd go further but can't work out what the opposite of pending is!

Reply:
Yeah I think you're right - the opposite of pending in this case is either `open` or `closed`, which is a bit yuk... I guess 'live' kind of fits but doesn't really encapsulate the closed state so much, so I'll maybe go with `not_pending && (in_future || online_q_a)` for clarity